### ISSUE TITLE
Load partials files only with handlebars's file extensions

### ISSUE THREAD

**Issue by 5ulo** [webdiscus/html-bundler-webpack-plugin#24]
## Feature request
In `loaderOptions` would be great to set for each `preprocessorOptions` to accept/ignore only specific extensions.

**What is motivation or use case for adding/changing the behavior?**

Imagine partials tree like this:
```
partials/
├─ card/
│  ├─ card.hbs
│  ├─ card.scss
├─ hero/
│  ├─ hero.hbs
│  ├─ hero.scss
```
As you can see `.hbs` files are organised next to its `.scss` file. Now if I include partials the native partials way like `{{>card/card}}` it renders the scss file instead of the hbs. If I rename the scss file to `card1.scss`, the hbs file is embedded correctly.
If I include partials with `{{include 'card/card'}}`, the hbs is included correctly.

**Describe the solution you'd like**

If the `preprocessorOptions` is limited for example to include only `.hbs` files, then would work also the classic partials `{{>card/card}}` include.

**Describe alternatives you've considered**

The only alternative is to use `{{include 'card/card'}}` which surprisingly works and ignores the `.scss` file.

Edit: `{{include ... }}` cannot be used inside loops as it wont passtrough data

**Comment by 5ulo** [comment]
Anyways.. this maybe would not be needed as a SASS partial file is named with a leading underscore. In that case the hbs partials are working as expected because files have different filenames `card` vs `_card`

**Comment by webdiscus** [comment]
Hello @5ulo,

thanks for issue report.
The bug is fixed in the `v2.2.1`.
Now you can have many files in a partial directory with the same basename, e.g.: 
```
-  card.hbs
-  card.js
-  card.scss
```

The partials `{{>card/card}}` is now correct included.

**Comment by 5ulo** [comment]
That was fast... and is working. Thanks
